lib/glific/third_party/gcs/gcs_worker.ex

```diff
@@ -116,11 +134,11 @@ defmodule Glific.GCS.GcsWorker do
   @doc """
     Queue urls for gcs jobs.
   """
-  @spec queue_urls(non_neg_integer, non_neg_integer, non_neg_integer) :: :ok
-  def queue_urls(organization_id, min_id, max_id) do
+  @spec queue_urls(non_neg_integer, non_neg_integer, non_neg_integer, String.t()) :: :ok
+  def queue_urls(organization_id, min_id, max_id, phase) do
     query =
       GCS.base_query(organization_id)
-      |> where([m], m.id > ^min_id and m.id <= ^max_id)
+      |> where([m], m.id >= ^min_id and m.id <= ^max_id)
```

Comment:
should we add condition to check GCS URL is null as well just to be double sure, in case we restart queue for an org